src/scenarios/init.ps1

```diff
@@ -1,24 +1,45 @@
 Param(
-    $Channel = 'master' # default is master
+    [string] $InstallDotnetFromChannel,
+    [string] $DotnetDirectory
 )
 
+# Add scripts and current directory to PYTHONPATH
 $scripts = Join-Path $PSScriptRoot '..\..\scripts' -Resolve
 $env:PYTHONPATH="$scripts;$PSScriptRoot"
-$dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
-$dotnetDirectory = Join-Path $PSScriptRoot '..\..\tools\dotnet\x64'
 
-Write-Host "Downloading dotnet from channel $Channel"
-python $dotnetScript install --channels $Channel -v
+If (($InstallDotnetFromChannel -ne "") -and ($DotnetDirectory -eq "")){
+    # Download dotnet from the specified channel
+    
+    $installDirectory= Join-Path $PSScriptRoot 'dotnet'
+    # Remove existing dotnet directory to make sure we only have one version of dotnet
+    If (Test-Path $installDirectory){
+        Write-Host "Removing $installDirectory ..."
+        Remove-Item  $installDirectory -Recurse
+    }
 
-if (!$?){
-    Write-Host "Dotnet installation failed."
-    Exit 1
+    Write-Host "Downloading dotnet from channel $InstallDotnetFromChannel"
+    $dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
+    python $dotnetScript install --channels $InstallDotnetFromChannel --install-dir $installDirectory
```

Comment:
I would not pass `--install-dir` here. Not passing it just uses the default location in the repo (which is `tools\dotnet\<arch>`. Having a separate copy here means a) it shows up as added files in git and b) we have random executable things that need cleanup outside of `tools` or `artifacts`.

Reply:
Cool I didn't know it can download directly to ```tools``` folder.